tsfm_public/models/tinytimemixer/utils/ttm_utils.py

```diff
@@ -176,7 +176,7 @@ def int_to_bool(value):
 
 
 # Utitlity: plot
-def plot_preds(trainer, dset, plot_dir, num_plots=10, plot_prefix="valid", channel=-1):
+def plot_preds(trainer, dset, plot_dir, num_plots=10, plot_prefix="valid", channel=-1, truncate_history=True):
```

Comment:
Eventually these utilities should be moved out of model-specific folders.

Reply:
Agree.

Reply:
Have these utilities been moved out to some other location?  The below code in ttm_tutorial.py is failing 

from tsfm_public.models.tinytimemixer.utils import (
    count_parameters,
    plot_preds,
)

Here are the errors. 
ImportError: cannot import name 'count_parameters' from 'tsfm_public.models.tinytimemixer.utils' (/content/tsfm/tsfm_public/models/tinytimemixer/utils/__init__.py)

ImportError: cannot import name 'plot_preds' from 'tsfm_public.models.tinytimemixer.utils' (/content/tsfm/tsfm_public/models/tinytimemixer/utils/__init__.py)

Reply:
@SatishGune They were moved -- let me double check.

Reply:
`plot_preds` is now `plot_predictions`. It can be imported with `from tsfm_public.toolkit.visualization import plot_predictions`. The signature of `plot_predictions` is now:
```
def plot_predictions(
    model: torch.nn.Module,
    dset: torch.utils.data.Dataset,
    plot_dir: str = None,
    num_plots: int = 10,
    plot_prefix: str = "valid",
    channel: int = -1,
    truncate_history: bool = True,
)
```
It takes a model as its first argument instead of a trainer. The `ttm_tutorial.ipynb` has been updated to reflect these changes.

Reply:
Thanks @wgifford . Much appreciated.